backend/src/openarchiefbeheer/destruction/api/serializers.py

```diff
@@ -124,7 +124,32 @@ def __init__(self, *args, **kwargs):
     def validate_assignees(
         self, assignees: list[DestructionListAssignee]
     ) -> list[DestructionListAssignee]:
+        current_assignee_pks = (
+            list(
+                self.instance.assignees.filter(role=ListRole.reviewer).values_list(
+                    "user__pk", flat=True
+                )
+            )
+            if self.instance
+            else []
+        )
         assignees_pks = [assignee["user"].pk for assignee in assignees]
+
+        if current_assignee_pks and current_assignee_pks != assignees_pks:
+            comment = str(self.initial_data.get("comment")).strip()
+
+            if not comment:
+                raise ValidationError(
+                    _("A comment should be provided when changing assignees.")
+                )
+
+            logevent.destruction_list_reassigned(
+                destruction_list=self.instance,
+                assignees=assignees,
+                comment=comment,
+                user=self.context["request"].user,
+            )
```

Comment:
At this point, the list has not been reassigned yet (just the data validation has happened). I would move this to the `update` method, after the list has been updated.